packages/core/src/sass/details-form.scss

```diff
@@ -8,6 +9,10 @@
         overflow-y: auto;
     }
 
+    &__scrollbar {
+        max-height: none;
```

Comment:
@mahdiaryayi-fs , can you move this to be applied only on `desktop` screens?
Because I'm not sure if `max-height: none` will work with `overflow-y: auto` rule applied in `mobile`.

Reply:
@zaki-binary Done!